src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.subsystems.climber;
+
+import edu.wpi.first.wpilibj.Relay;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Climber extends SubsystemBase {
+
+  private final ClimberIO climberIO;
+  private final ClimberIOInputsAutoLogged climberIOInputs = new ClimberIOInputsAutoLogged();
+  private final Relay climberLock = new Relay(0); // TODO: Add constant for channel
+  // TODO: Add Soloenoid
+
+  public Climber(ClimberIO climberIO) {
+    super();
+
+    this.climberIO = climberIO;
+
+    climberIO.updateInput(climberIOInputs);
+  }
+
+  public Command raise(double percentOutput) {
+    return Commands.run(
+        () -> {
+          climberIO.setMotorOutputPercent(percentOutput);
+        },
+        this);
+  }
+
+  public Command lower(double percentOutput) {
+    return Commands.run(
+        () -> {
+          climberIO.setMotorOutputPercent(percentOutput);
+        },
+        this);
+  }
```

Comment:
raise() and lower() are the same right now.

Consider one of these:
 - Combine them into a single method.
 - Have lower() send a negative output percent to the motor, and raise() send a positive voltage. They both would need to take a positive number.